src/main/java/com/questhelper/steps/DetailedQuestStep.java

```diff
@@ -374,23 +382,24 @@ public void renderArrow(Graphics2D graphics)
 				return;
 			}
 
-			LocalPoint lp = QuestPerspective.getInstanceLocalPointFromReal(client, worldPoint);
-			if (lp == null)
-			{
-				return;
-			}
+			List<LocalPoint> localPoints = QuestPerspective.getInstanceLocalPointFromReal(client, worldPoint);
+			if (localPoints.isEmpty()) return;
```

Comment:
This check seems superfluous, unless it's faster than letting Java just not loop on line 388